src/lms-ds-installer/pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "lms-ds-installer"
+version = "0.1.0"
+description = "Learing Management System Data Store (LMS-DS) Installer"
+authors = ["Ed-Fi Alliance, LLC and Contributors"]
+

```

Comment:
```suggestion
license = "Apache-2.0"

```

Reply:
You can do that in a pyproject.toml file? Nice.

Reply:
Done